test/gurka/test_landmarks.cc

```diff
@@ -13,15 +13,46 @@ using namespace valhalla::mjolnir;
 const std::string db_name = "landmarks.db";
 const std::filesystem::path file_path = db_name;
 
+namespace {
+valhalla::gurka::map BuildPBF(const std::string& workdir) {
+  const std::string ascii_map = R"(
+      A-------B------C
+    )";
+
+  const gurka::nodes nodes = {
+      {"A", {{"name", ""}, {"amenity", "university"}}},
+      {"B", {{"name", "hai di lao"}, {"amenity", "restaurant"}}},
+      {"C", {{"name", "ke ji lu"}, {"amenity", ""}}},
```

Comment:
IMO this should be a degenerate case we want to throw early, as it has no meaning without a type.

Reply:
let's also include a node that shouldn't be saved. Maybe we can make this more realistic by e.g.

```
  const std::string ascii_map = R"(
      a-------b------c
          A    B     C
    )";
```
So A, B, C would be on the side of the road, and a, b, c are simply graph nodes without any tags (which shouldn't appear in the db, but I think do currently, see above).

Reply:
good example I will update that.